test/simple/simple.status

```diff
@@ -9,12 +8,6 @@ test-microtask-queue-run-domain   : PASS,FLAKY
 [$system==win32]
 test-timers-first-fire            : PASS,FLAKY
 
-[$system==linux]
```

Comment:
Maybe better to leave the section headers? Otherwise it's not so obvious how to use these status files.